src/app/tests/suites/certification/Test_TC_SMCO_2_4.yaml

```diff
@@ -0,0 +1,587 @@
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for End of Service Alert Test Event"
+      PICS: DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_END_OF_SERVICE_ALERT
+
+    - label:
+          "TH waits for a report of EndOfServiceAlert attribute from DUT with a
+          timeout of 300 seconds"
+      PICS: SMOKECO.S.A0007
+      command: "waitForReport"
+      attribute: "EndOfServiceAlert"
+      timeout: 300
+      response:
+          value: 1
+          constraints:
+              type: enum8
+
+    - label: "TH reads ExpressedState attribute from DUT"
+      PICS: SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          value: 6
+          constraints:
+              type: enum8
+
+    - label: "TH reads EndOfService event from DUT"
+      PICS: SMOKECO.S.E04
+      command: "readEvent"
+      event: "EndOfService"
+      EVENT_NUMBER: EVENT_NUMBER + 6
+      response:
+          value: {}
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for End of Service Alert Test Event
+          Clear"
+      PICS: DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_END_OF_SERVICE_ALERT_CLEAR
+
+    - label:
+          "TH waits for a report of EndOfServiceAlert attribute from DUT with a
+          timeout of 300 seconds"
+      PICS: SMOKECO.S.A0007
+      command: "waitForReport"
+      attribute: "EndOfServiceAlert"
+      timeout: 300
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label: "TH reads ExpressedState attribute from DUT"
+      PICS: SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label: "TH reads AllClear event from DUT"
+      PICS: SMOKECO.S.E0a
+      command: "readEvent"
+      event: "AllClear"
+      EVENT_NUMBER: EVENT_NUMBER + 7
+      response:
+          value: {}
+
+    - label: "TH reads TestInProgress attribute from DUT"
+      PICS: SMOKECO.S.A0005
+      command: "readAttribute"
```

Comment:
Test plan (step 35, right?) has a subscribe here....

Reply:
OK, it's changed.